pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -71,6 +72,12 @@ class Agent(Generic[AgentDeps, ResultData]):
 
     If `None`, we try to infer the agent name from the call frame when the agent is first run.
     """
+    end_strategy: EndStrategy
+    """The strategy for handling multiple tool calls when a final result is found.
+
+    - "correct": Process all tool calls even after finding a final result
+    - "early": Stop processing other tool calls once a final result is found
+    """
```

Comment:
If we put these docs on `EndStrategy`, we don't have to repeat them, mkdocstrings will take care of creating a link to `EndStrategy` wherever it's used.

I don't really like `'correct'` - it makes the other thing seem incorrect. Perhaps `'complete', `'call-all-tools'` or `'exhaustive'`?